frontend/src/lib/components/Popup/Popup.tsx

```diff
@@ -70,168 +70,166 @@ let nestedPopupReceivedClick = false
  *
  * Often used with buttons for various menu. If this is your intention, use `LemonButtonWithPopup`.
  */
-export const Popup = React.forwardRef<HTMLDivElement, PopupProps>(
-    (
-        {
-            children,
-            referenceElement,
-            overlay,
-            visible,
-            onClickOutside,
-            onClickInside,
-            placement = 'bottom-start',
-            fallbackPlacements = ['bottom-start', 'bottom-end', 'top-start', 'top-end'],
-            className,
-            actionable = false,
-            middleware,
-            sameWidth = false,
-            maxContentWidth = false,
-            additionalRefs = [],
-            closeParentPopupOnClickInside = false,
-            style,
-            getPopupContainer,
-            showArrow,
-        },
-        ref
-    ): JSX.Element => {
-        const popupId = useMemo(() => uniqueMemoizedIndex++, [])
-        const parentPopupId = useContext(PopupContext)
+export const Popup = React.forwardRef<HTMLDivElement, PopupProps>(function Popup(
```

Comment:
For the inner pre-forwardRef component we often follow a `ComponentInternal` ('PopupInternal` here) style – it's clearer in stack traces

Reply:
Thanks, that makes sense! Updated.